presentation/screenshots.py

```
@@ -0,0 +1,67 @@
+
+output_dir = Path("presentation/images/")
+output_dir.mkdir(exist_ok=True)
+
+# Use MacBook Pro's native resolution and scale factor 2 for Retina
+viewport = {"width": 2940, "height": 1654}
+scale = 2  # high-resolution (Retina)
+
+with sync_playwright() as p:
+    browser = p.chromium.launch()
+    context = browser.new_context(viewport=viewport, device_scale_factor=scale)
+    for url, fname in zip(urls, filenames):
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)